controllers/admin/AdminImportController.php

```diff
@@ -118,47 +118,106 @@ public function __construct()
                 );
 
                 $this->available_fields = array(
-                    'no' => array('label' => $this->trans('Ignore this column', array(), 'Admin.Advparameters.Feature')),
-                    'id_product' => array('label' => $this->trans('Product ID', array(), 'Admin.Advparameters.Feature')),
-                    'product_reference' => array('label' => $this->trans('Product Reference', array(), 'Admin.Advparameters.Feature')),
+                    'no' => array(
+                        'label' => $this->trans('Ignore this column', array(), 'Admin.Advparameters.Feature'),
+                    ),
+                    'id_product' => array(
+                        'label' => $this->trans('Product ID', array(), 'Admin.Advparameters.Feature'),
+                    ),
+                    'product_reference' => array(
+                        'label' => $this->trans('Product Reference', array(), 'Admin.Advparameters.Feature'),
+                    ),
                     'group' => array(
-                        'label' => $this->trans('Attribute (Name:Type:Position)', array(), 'Admin.Advparameters.Feature').'*'
+                        'label' => $this->trans(
+                            'Attribute (Name:Type:Position)',
+                            array(),
+                            'Admin.Advparameters.Feature'
+                        ) . '*',
```

Comment:
Seems like there is no rule in the code (not in core nor modules). Core team opinion seems unclear too. And despite SF standard (no space usage), SF community seems divided too... SF PGP CS Fixer seems kinda buggy as soon as you concatenate on multiple lines (indentation adds spaces before or after the dot (`.`).
And PSR are just ignoring this case.
Well, I guess we have no rules settled yet...